ls1intum/Artemis
# `General`: Use same height if no statistics are available in course card

### Checklist
#### General


#### Client

### Motivation and Context


Fixes a styling issue on the start page of Artemis (see screenshots)

### Steps for Testing

Prerequisites:
1. Log in to Artemis
2. Have/create two courses, one has graded exercises the other one does not (may be completely empty)
3. Check that the height of the course cards is the same on the start page

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Old styling:
![Bildschirmfoto 2021-10-26 um 15 21 54](https://user-images.githubusercontent.com/38322605/138887981-2059d2d8-0211-4e2c-a28d-866c20b5f312.png)
Correct styling:
![Bildschirmfoto 2021-10-26 um 15 22 09](https://user-images.githubusercontent.com/38322605/138887977-0a57e9ed-5a62-488a-92e3-79ca31e39d9d.png)
